.github/workflows/ci.yml

```diff
@@ -78,6 +79,7 @@ jobs:
               - barretenberg/cpp/pil/**
               - barretenberg/cpp/src/barretenberg/vm/**
               - barretenberg/cpp/src/barretenberg/**/generated/*
+              - barretenberg/cpp/src/barretenberg/client_ivc.{hpp,cpp}
```

Comment:
This change will actually stay. It just says run e2e if client_ivc changes which is a no-brainer